sparse_ho/tests/test_svm.py

```diff
@@ -14,71 +14,66 @@
 from sparse_ho.ho import grad_search
 from sparse_ho.criterion import HeldOutSmoothedHinge
 from sparse_ho.utils import Monitor
+
+
 n_samples = 100
 n_features = 300
 
-X_train, y_train = datasets.make_classification(
+X, y = datasets.make_classification(
     n_samples=n_samples,
     n_features=n_features, n_informative=50,
     random_state=11, flip_y=0.1, n_redundant=0)
-X_train_s = csr_matrix(X_train)
-
-
-X_val, y_val = datasets.make_classification(
-    n_samples=n_samples,
-    n_features=n_features, n_informative=50,
-    random_state=12, flip_y=0.1, n_redundant=0)
-X_val_s = csr_matrix(X_val)
+X_s = csr_matrix(X)
 
 
-y_train[y_train == 0.0] = -1.0
-y_val[y_val == 0.0] = -1.0
+y[y == 0.0] = -1.0
+idx_train = np.arange(0, 50)
+idx_val = np.arange(50, 100)
 
 
-C = 0.001
+C = 0.01
 log_C = np.log(C)
 tol = 1e-16
 
 models = [
-    SVM(
-        X_train, y_train, log_C, max_iter=10000, tol=tol),
-    SVM(
-        X_train_s, y_train, log_C, max_iter=10000, tol=tol)
+    SVM(log_C, max_iter=10000, tol=tol),
```

Comment:
@QB3 I think this is no longer needed and you need to parametrize your test with X, Xs instead